katacomb/scripts/continuum_pipeline.py

```diff
@@ -193,13 +193,20 @@ def main():
     post_process_args(args, katdata)
 
     uvblavg_args, mfimage_args, band = _infer_defaults_from_katdal(katdata)
-
+    # Get frequncies and convert them to MHz
+    freqs = katdata.freqs/1e6
     # Get config defaults for uvblavg and mfimage and merge user supplied ones
-    uvblavg_parm_file = pjoin(CONFIG, f'uvblavg_MKAT_{band}.yaml')
-    log.info('UVBlAvg parameter file for %s-band: %s', band, uvblavg_parm_file)
-    mfimage_parm_file = pjoin(CONFIG, f'mfimage_MKAT_{band}.yaml')
-    log.info('MFImage parameter file for %s-band: %s', band, mfimage_parm_file)
-
+    # Check if the observation is narrowband by checking the bandwith size
+    if (freqs[-1] - freqs[0]) < 200:
+        uvblavg_parm_file = pjoin(CONFIG, f'uvblavg_MKAT_narrow_{band}.yaml')
```

Comment:
This might cause problems if anyone erroneously tries to do the narrowband imaging on any band that isn't L-band, because for example currently 'uvblavg_MKAT_narrow_S.yaml' doesn't exist. 
I think the simplest solution here is to check the band is 'L' if yes  then proceed, if not still use the 'L' band parameter file but warn the user that this is untested.

Reply:
The assumption behind that was narrow band observations are only done in the L-band. However, the catch makes sense and i have fixed that to make it more general.